app/services/api.ts

```
@@ -0,0 +1,64 @@
+        'Content-Type': 'application/json',
+    },
+});
+
+export const userService = {
+    // Get all users
+    getUsers: async () => {
+        try {
+            const response = await api.get('/users');
+            return response.data;
+        } catch (error) {
+            throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)